<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.nationalguard.fitness.FitnessCal" tests="1" time="0.013" ignored="0" failures="0" timestamp="14 Jan 2020 00:35:55 GMT" skipped="0" hostname="ajay-PC" errors="1">
  <testcase name="fitnessCalculatorMale" time="0.013" classname="com.nationalguard.fitness.FitnessCal">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: fitnessCalculatorMale([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)Chrome,null,null]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: fitnessCalculatorMale([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)Chrome,null,null]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:52)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1278)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1171)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- fitnessCalculatorMale -->
</testsuite> <!-- com.nationalguard.fitness.FitnessCal -->
